lib/events.js

```diff
@@ -1,74 +1,82 @@
-const common = require('./common.js')
-const httph = require('./http_helper.js')
-const query = require('./query.js')
-const assert = require('assert')
-const fs = require('fs')
+const common = require('./common.js');
+const httph = require('./http_helper.js');
 const logs = require('./log-drains.js');
 
-const select_build_by_foreign_key = query.bind(query, fs.readFileSync("./sql/select_build_by_foreign_key.sql").toString('utf8'), (r) => { return r });
+// eslint-disable-next-line max-len
+// const select_build_by_foreign_key = query.bind(query, fs.readFileSync('./sql/select_build_by_foreign_key.sql').toString('utf8'), (r) => r);
 
-async function create(pg_pool, req, res, regex) {
-  let payload = await httph.buffer_json(req)
-  if(!payload.action || !payload.key) {
-    throw new common.BadRequestError()
+async function create(pg_pool, req, res /* regex */) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function 'create' has a complexity of 14. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)